cutlass/cf.go

```diff
@@ -493,6 +507,8 @@ func (a *App) DownloadDroplet(path string) error {
 
 func (a *App) Destroy() error {
 	if a.logCmd != nil && a.logCmd.Process != nil {
+		a.logCmd = nil
+		a.Stdout = nil
 		if err := a.logCmd.Process.Kill(); err != nil {
```

Comment:
Accessing `a.logCmd` would be a nil pointer dereference, right?

Reply:
The line immediately before this checks that `a.logCmd != nil`. I think we're safe to set that here.

Reply:
@ryanmoran I was referring to line `512` on which I posted the comment.
It'll end up as `nil.Process.Kill()` due to `a.logCmd` set to `nil` two lines up.

Reply:
oh jeez. what the hell.

Reply:
ok. moved to after the `Process.Kill`